docs/docs/channels/email/mandrill.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 11
+---
+
+# Mandrill
+
+You can use the [Mandrill](https://mandrillapp.com/) provider to send transactional emails to your customers using the Novu Platform with a single API.
+
+## Getting Started
+
+To use the Mandrill provider, you will need to create a Mandrill account and add your API key to the Mandrill integration on the Novu platform.
+
+## Generate API Key
+
+To generate a new API key in Mandrill, you can follow these steps:
+
+- [Sign up](https://login.mailchimp.com/signup/) or [Log in](https://login.mailchimp.com/) to your Mandrill account.
```

Comment:
Could you add a little comment in the preface regarding Mandrill being an add-on for Mailchimp, so if people follow links don't get surprised or think they are wrongly redirected?

Reply:
how about in your intro text. I use Mandrill by MailChimp?

Reply:
Yes, I meant in the intro!

Reply:
To be clear, this is what I meant. Let me know your thoughts
`You can use the [Mandrill by Mailchimp](https://mandrillapp.com/) provider to send transactional emails to your customers using the Novu Platform with a single API.`

Reply:
Go for it.